docs/diagnostics/PX1064.md

```diff
@@ -0,0 +1,97 @@
+# PX1064
+This document describes the PX1064 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                         | Type                           | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------------------- | ------------------------------ | ----------- | 
+| PX1064 | The declaration of the exception class introduces new serializable fields but does not declare an override for the `GetObjectData` method. This will cause an incorrect deserialization of the exception data and will lead to runtime errors in Acumatica ERP. | Error | Available | 
+
+## Diagnostic Description
+
+Every class that that is derived from the System.Exception class (including PXException-derived classes) that declares new serializable data always provide an override for the `GetObjectData` method.
+
+By serializable data, the diagnostic understands any new data declared in the exception class that will be serialized by .Net serializers and Acumatica reflection based serializer which are the following:
+ - New fields if they are not marked with `System.NonSerializedAttribute` attribute:
+ - New auto generated properties if they are not marked with `System.NonSerializedAttribute` attribute:
+For example, the following types of members are not considered serializable:
+ - Static fields and constants
+ - Static properties
+ - Calculated properties
+ - Properties with explicit backing field
```

Comment:
The examples I provided for PX1063 should be copied here as well:
- https://github.com/Acumatica/Acuminator/pull/458/files#r1023533275
- https://github.com/Acumatica/Acuminator/pull/458/files#r1023533596